x.ps1

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env pwsh
+
+# See x.sh for why these scripts exist.
+
+$xpy = $(Split-Path $MyInvocation.MyCommand.Path -Parent) + "/x.py"
```

Comment:
I think [`Join-Path`](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/join-path?view=powershell-7.2) is what you want here, then you wouldn't need the splat on the next line (although I'm not sure why you need it as-is either)

Reply:
I need the splat because `x.py` is not actually the command being executed (that defeats the whole point of the scripts). It needs to be part of the argument list so I can pass it to the `python3` binary.